core/arch/arm/kernel/thread_private.h

```diff
@@ -163,8 +163,6 @@ void thread_init_vbar(void);
 /* Handles a stdcall, r0-r7 holds the parameters */
 void thread_std_smc_entry(void);
 
-struct thread_core_local *thread_get_core_local(void);
```

Comment:
This change should go into the "core: arm: kernel: make thread_core_local public" commit

Reply:
Thanks @jenswi-linaro. I'll fix this, then add your tags.